setup/Core/UpdateLocalizationFilesTask.cs

```diff
@@ -1,38 +1,33 @@
 ﻿using System.Diagnostics;
-using System.IO;
-using System.Windows;
+using Terraria.ModLoader.Setup.Core.Abstractions;
 
-namespace Terraria.ModLoader.Setup;
+namespace Terraria.ModLoader.Setup.Core;
 
-internal class UpdateLocalizationFilesTask : SetupOperation
+public sealed class UpdateLocalizationFilesTask : SetupOperation
 {
-	private const string UpdateLocalizationFilesPath = $"solutions/UpdateLocalizationFiles.py";
+	private const string UpdateLocalizationFilesPath = "solutions/UpdateLocalizationFiles.py";
 
-	public UpdateLocalizationFilesTask(ITaskInterface taskInterface) : base(taskInterface)
+	public override async Task Run(IProgress progress, CancellationToken cancellationToken = default)
 	{
-	}
+		using var taskProgress = progress.StartTask("Updating localization files...");
```

Comment:
Yeah, if anything writes to stdout not using `ITaskProgress` while the live display is rendered, it will bug out. There is not much I can do about that. Maybe we should just disable live progress for this task.